student.json

```diff
@@ -0,0 +1,40 @@
+{
+	"student":
+	{
+			"fist_name": "John",
+			"age": 21,
+			"driver_licence": null,
+			"married": false,
+			"friends": ["Jane", 21, true, false,"William",20, false, false],
```

Comment:
This field `friends` is array and JSON is not guarantee that the order of items will be the same as in text representation. So better to use array of objects or simplify structure to use only names(strings) in array. It's not recommended to use different types of data in the same array

Reply:
Now it's clear for me. Thank you for clarificaition!